.github/workflows/ai-runner-live-pipelines-docker.yaml

```
@@ -39,7 +39,7 @@ jobs:
         uses: tj-actions/changed-files@823fcebdb31bb35fdf2229d9f769b400309430d0 # v46.0.3
         with:
           files: |
-            runner/docker/Dockerfile.live-base
+            runner/docker/Dockerfile.live-base*
```

@pschroedl I don't think we need this change
```suggestion
            runner/docker/Dockerfile.live-base
```